src/main/java/edu/iis/powp/adapter/MyAdapter.java

```diff
@@ -9,6 +10,8 @@
  * Plotter adapter to drawer with several bugs.
  */
 public class MyAdapter extends DrawPanelController implements IPlotter {

```

Comment:
Name unchanged. Inheritance is not used (no super drawLine calls). Should be removed!